File: java/src/main/java/com/iteso/nintendo/attacks/WaterGun.java
package com.iteso.nintendo.attacks;

public class WaterGun implements Attack {
    static int damage;
    static String name = "Water Gun";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/WaterGun.py
class WaterGun:
    def __init__(self):
        self.damage = None
        self.name = 'Water Gun'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
